README.md

```diff
@@ -94,7 +94,7 @@ sudo apt-get install make gcc g++ bc python xutils-dev bison flex libgcrypt20-de
 
 2. Clone the SGX-LKL git repository:
 ```
-git clone --branch oe_port git@github.com:lsds/sgx-lkl.git
+git clone --branch oe_port --recursive --progress git@github.com:lsds/sgx-lkl.git
```

Comment:
Using the ssh url will fail if you don't have permissions to the repo. Let's switch to http.
Also, `--progress` is not useful here, see https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---progress.

Reply:
Done